core/includes/bootstrap.inc

```diff
@@ -1750,6 +1751,13 @@ function backdrop_serve_page_from_cache(stdClass $cache) {
     return;
   }
 
+  // Handle a loose match based on date only, if no Etag was provided.
+  if ($if_modified_since && !$if_none_match && $if_modified_since <= $cache->created) {
```

Comment:
Hm... `$if_modified_since <= $cache->created` - shouldn't that be `>=`?
The current logic means to me:
If the page has changed _before_ my previous request... - it has in my previous request.

To my understanding what we want here is: "If the page has changed *since* my previous request*, hence `>= $cache->created`